src/components/connection-modal/connection-modal.css

```diff
@@ -243,11 +243,11 @@
 }
 
 [dir="ltr"] .scratch-link-help-step {
-    margin-left: 3rem;
+    margin-left: 2.5rem;
```

Comment:
adjusted these so the help step numbers would have the same resulting margin as before